code-for-canada/project-thundercat
# [Dashboard] Creating dashboard table with temporary row

# Description

This PR is introducing the new table in dashboard page. I also added a temporary row that shows the global view of this future table.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

![image](https://user-images.githubusercontent.com/23021242/59525461-61512780-8ea4-11e9-8c6d-600293866e7d.png)

# Testing

Manual steps to reproduce this functionality:

1.  Login to the system.
2.  Make sure that the dashboard page displays the expected result.

# Checklist

Applicable for all code changes.
